templates/octaviaapi/config/octavia.conf

```diff
@@ -102,7 +102,7 @@ auth_type = password
 project_domain_name = Default
 user_domain_name = default
 region_name = regionOne
-username = neutron
+username = {{ .ServiceUser }}
```

Comment:
ditto. I don't think we need the credential in [neutron] section at least as of zed.